recipes/dotty-dict/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dotty-dict" %}
+{% set version = "1.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
The url for pypi uses an underscore, so `dotty_dict` (e.g. https://pypi.io/packages/source/d/dotty_dict/dotty_dict-1.3.0.tar.gz) . I would just add another jinja variable called something like `pypi_name` and use that in the url.